File: src/score.py
```python
class Score:
    def __init__(self):
        self.score = 0

    def updateScore(self, increment):
        self.score += increment

    def decreaseScore(self, decrement):
        self.score -= decrement
        if self.score < 0:
            self.score = 0

    def getScore(self):
        return self.score

    def resetScore(self):
        self.score = 0
```

File: ColorDash/src/main/java/com/colordash/game/Score.java
```java
package com.colordash.game;

public class Score {
    private int score;

    public Score() {
        this.score = 0;
    }

    public int getScore() {
        return this.score;
    }

    public void increaseScore(int value) {
        this.score += value;
    }

    public void resetScore() {
        this.score = 0;
    }
}
```